src/components/Header/index.ts

```diff
@@ -0,0 +1 @@
+export { Header } from './Header';
```

Comment:
I don't think there was any need to create separate components. Especially as the code isn't that big.

Reply:
Header had 242L of code, I don't believe having large components is a good practice. Composition is a much better solution IMO. But you're free to disagree.